django/contrib/admindocs/views.py

```diff
@@ -206,7 +205,7 @@ def get_context_data(self, **kwargs):
         for field in opts.fields:
             # ForeignKey is a special case since the field will actually be a
             # descriptor that returns the other object
-            if isinstance(field, models.ForeignKey):
+            if field.many_to_one:
```

Comment:
This check will pass for a GenericForeignKey, but there is no remote_field in GFK (nor can there be one, as GenericForeignKey's remote field should be every primary key in all models). We need a field.remote_field is not None check here and likely in many other places, too.